source/clear-linux/tutorials/redis.rst

```diff
@@ -0,0 +1,254 @@
+.. _redis:
+
+Run Redis on |CL-ATTR|
+######################
+
+`redis` is an in-memory key:value store, designed for quick lookups,
+accessed over the network. In this tutorial, you'll install redis and launch
+a redis-server on |CL|, plus learn a few basic commands. Also, we invite
+you to pull our `Clear Linux Redis instance`_ on dockerhub\* for application
+or infrastructure development.
+
+Redis can be used as a NoSQL database for web applications. Yet it's also
+easily integrated with an existing stack, for refreshing images on a website,
+cache management, real-time analysis, or as a leaderboard. Redis offers many
+client libraries with language-specific bindings like Python, Perl, Ruby,
+and more. Explore the development possibilities on the `redis website`_.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Prerequisites
+*************
+* Install the `redis-native` on |CL|
+* Install `cloud-control` on |CL| (only required in Example 2)
+
+Install the redis bundle
+************************
+
+In |CL|, find redis in the `redis-native` bundle.
+
+#. Open a Terminal and login as an administrative user.
+
+#. Add :file:`redis-native`.
+
+.. code-block:: bash
+
+   sudo swupd bundle-add redis-native
+
+.. note::
+
+   If the bundle already exists, no action is required.
+
+Launch the redis-server
+***********************
+
+Verify the redis-service is running. By default it runs on port 6379.
+
+#. A systemd service is available. Launch the service.
+
+   .. code-block:: bash
+
+      systemctl start redis
+
+   .. note::
+
+      To stop redis run :command:`systemctl stop redis`.
+
+#. Run the status command to assure the service is running:
```

Comment:
This line wording "Run the status command" better aligns wit hthe `systemctl status redis` step. I'm guessing this got shuffled during editing :)